### PR TITLE
Hw 12

### PR DESCRIPTION
**Pull request by eltrubetskaya**
Create installer for module from the Lesson 6 (“AskQuestion” module)
Create model / resource model / collection
Implement saving data to the database in your frontend controller
Create new listing for this table (see Lesson 10) with filters
Show questions to this product in the tab on the Product Page
Advanced: create MassAction to change status from “Pending” (red) to “Answered” (green)

### CODE REVIEW

File: app/code/ElTrubetskaia/AskQuestion/view/frontend/templates/product/view/question_list.phtml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php /** @var \ElTrubetskaia\AskQuestion\Block\Question $block */
+$askQuestions = $block->getAskQuestions();
+?>
+<?php if ($askQuestions !== null) : ?>
+    <ul>
+        <?php /** @var \ElTrubetskaia\AskQuestion\Model\AskQuestion $askQuestion */
+        foreach ($askQuestions as $askQuestion) :
+            ?>
+            <?php if ($askQuestion->getSku() === $block->getCurrentProduct()->getSku()) : ?>
```
Comment (chabanenk0) [inline app/code/ElTrubetskaia/AskQuestion/view/frontend/templates/product/view/question_list.phtml]: This condition should be checked on database side. Please create \ElTrubetskaia\AskQuestion\Block\Question::getAskQuestionsForProduct(...)
and pass it to collection (addFieldToFilter)

File: app/code/ElTrubetskaia/AskQuestion/view/adminhtml/layout/askquestion_page_index.xml [hunk 2]
```diff
@@ -1,12 +1,11 @@
 <?xml version="1.0"?>
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <head>
+        <css src="ElTrubetskaia_AskQuestion::css/customClasses.css"/>
```
Comment (chabanenk0) [inline app/code/ElTrubetskaia/AskQuestion/view/adminhtml/layout/askquestion_page_index.xml:4]: Don't include custom styles, add your styles to: ...<Namespace>_<Module>/web/css/source/_module.less

File: app/code/ElTrubetskaia/AskQuestion/view/frontend/templates/product/view/question_list.phtml (hunk 1 above)
Comment (eltrubetskaya) [reply app/code/ElTrubetskaia/AskQuestion/view/frontend/templates/product/view/question_list.phtml]: fixed. thanks!

File: app/code/ElTrubetskaia/AskQuestion/view/adminhtml/layout/askquestion_page_index.xml (hunk 2 above)
Comment (eltrubetskaya) [reply app/code/ElTrubetskaia/AskQuestion/view/adminhtml/layout/askquestion_page_index.xml:4]: If it's styles for adminhtml - should create a new theme for AdminPanel and then in app/design/adminhtml/.../web/css/source/_module.less apply new styles?

Comment (chabanenk0) [reply app/code/ElTrubetskaia/AskQuestion/view/adminhtml/layout/askquestion_page_index.xml:4]: sorry, didn't mentioned that it's admin. Ok

**Comment by chabanenk0** [comment]
@eltrubetskaya Approved, but why this PR is not to master (or develop) but to hw-10?

**Comment by eltrubetskaya** [comment]
@chabanenk0 because parent branch hw-12 was branch hw-10, and  this PR to show diff

**Pull request merged**